Gems/AtomLyIntegration/CommonFeatures/Code/Source/Scripting/EditorEntityReferenceComponent.cpp

```diff
@@ -26,7 +26,7 @@ namespace AZ
                     editContext->Class<EditorEntityReferenceComponent>(
                         "Entity Reference", "Contains a reference list to other entities")
                         ->ClassElement(Edit::ClassElements::EditorData, "")
-                        ->Attribute(Edit::Attributes::Category, "Atom")
```

Comment:
How is this actually used? It seems like something that's not rendering specific...does this belong in Atom at all?

Reply:
@HogJonny-AMZN

Reply:
It doesn't seem to have much use, other than it's a generic component that holds a list of entities. I noticed it is being used in SliceStabilityTestFramework. This component has an EBUs but it is not exposed to scripting. I'll open a GHI to find a better place for it (or remove it). It was written by @hershey5045 and I guess that's why it was  under the "Atom" category.

Reply:
Moved it to the existing "Miscellaneous" category